### ISSUE TITLE
sophomorix-passwd switch -t and -s not working

### ISSUE THREAD

**Issue by baumhof** [linuxmuster/sophomorix4#20]
I have sophomorix-samba Version 3.50.3-1

If i use -s (all Students) or -t (all teachers) it complains i "made a mistake, when specifying options".

I guess its not implemented yet.

LG

Holger

**Comment by jeffbeck** [comment]
Yes thats true. Im working on an improved selection of users using sophomorixRole.
